### ISSUE TITLE
no-block-empty and empty styled-components

### ISSUE THREAD

**Issue by rifler** [stylelint/stylelint#3494]
> What is the problem you're trying to solve?

For some reasons I use empty styled components like `const DeleteIcon = styled(IconCross)``;`. Stylelint shows error when `no-block-empty` is enabled and this sounds logical

> What solution would you like to see?

It will be convenient for me to have an opportunity to disable this rule only for components, e.g.:
good:
```js
const DeleteIcon = styled(IconCross)``;
```

bad:
```js
const DeleteIcon = styled(IconCross)`
   ...some rules...
   // error
   div {}
`;
```

ps: thank you for awesome tool

**Comment by jeddy3** [comment]
@rifler Thanks for the request and for using the template. 

It's an interesting request. I'm not exactly sure what can be done. The parses treats styled-components as the block of CSS rule-sets. It must convert it into this format for the built-in rules to work.

The recommendation has always been to use [disable commands](https://github.com/stylelint/stylelint/blob/master/docs/user-guide/configuration.md#turning-rules-off-from-within-your-css) when a user wishes to turn off a turn for a particular snippet of code. It feels like this is still appropriate advice, rather than trying to add an exception into this particular rule. 

```js
const DeleteIcon = styled(IconCross)`/* stylelint-disable no-empty-block */`;
```

Does anyone else have any thoughts on this?

**Comment by rifler** [comment]
Thanks for quick response
My case is rare and I can easily add ignore comments, so I close issue
